tests/wpt/meta-legacy-layout/gamepad/idlharness.window.js.ini

```diff
@@ -1,28 +1,4 @@
 [idlharness.window.html]
-  [Gamepad interface: attribute vibrationActuator]
-    expected: FAIL
-
-  [GamepadHapticActuator interface: existence and properties of interface object]
-    expected: FAIL
-
-  [GamepadHapticActuator interface object length]
-    expected: FAIL
-
-  [GamepadHapticActuator interface object name]
-    expected: FAIL
-
-  [GamepadHapticActuator interface: existence and properties of interface prototype object]
-    expected: FAIL
-
-  [GamepadHapticActuator interface: existence and properties of interface prototype object's "constructor" property]
-    expected: FAIL
-
-  [GamepadHapticActuator interface: existence and properties of interface prototype object's @@unscopables property]
-    expected: FAIL
-
-  [GamepadHapticActuator interface: attribute effects]
-    expected: FAIL
-
   [GamepadHapticActuator interface: operation playEffect(GamepadHapticEffectType, optional GamepadEffectParameters)]
```

Comment:
I think we can fix these as well, as per my comments in zulip re setting up the promise.

Reply:
Actually, we can't.